src/ontology/mondo-edit.obo

```diff
@@ -437627,6 +437629,13 @@ synonym: "small lungs" RELATED [PMID:24836585]
 is_a: MONDO:0005275 {source="PMID:24836585", source="https://orcid.org/0000-0003-0113-912X"} ! lung disease
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700003
+name: obstetric disorder
+def: "Disorder associated with pregnancy, childbirth, and puerperium." [https://orcid.org/0000-0002-4142-7153]
+is_a: MONDO:0000001 ! disease or disorder
```

Comment:
please add an equiv axiom (Equivalent To)

'pregnancy disorder' or 'puerperal disorder'

Reply:
done